test/test_nn.py

```diff
@@ -3295,6 +3295,7 @@ def perm_fn(x):
             self.assertEqual(tuple(result.shape), tuple(ref_output.shape))
             np.testing.assert_allclose(result, ref_output, atol=1e-5)
 
+    @set_default_dtype(torch.double)
```

Comment:
Why is this here?
Bad merge after https://github.com/pytorch/pytorch/pull/107069 ?
cc @kurtamohler

Reply:
The PR #107069 missed to add the decorator for this test case which makes it run with float type while based on the tolerance defined in the test case it should be double. Same decorator have been added for lots of other test cases in the same test file with the PR, while this one seems missed.

Reply:
Thanks for catching that!